M2/Macaulay2/m2/matrix.m2

```diff
@@ -85,6 +85,8 @@ Matrix * Number :=
 Matrix * RingElement := (m,r) -> (
     if ring r =!= ring m then try r = promote(r,ring m) else m = promote(m,ring r);
      map(target m, source m, reduce(target m, raw m * raw r)))
+Matrix / Number      :=
+Matrix / RingElement := (m,r) -> 1/r * m
```

Comment:
I think this is intentionally not defined:
```m2
i3 : R = kk[x,y]

o3 = R

o3 : PolynomialRing

i4 : matrix{{x}} / x

o4 = | 1 |

                    1             1
o4 : Matrix (frac R)  <-- (frac R)
```

Reply:
But this is defined:
```m2
i3 : matrix{{x}} // x

o3 = | 1 |

             1      1
o3 : Matrix R  <-- R
```

Reply:
Even over integers:
```m2
i2 : Matrix / Number      := (m,r) -> 1/r * m

o2 = FunctionClosure[stdio:2:30-2:39]

o2 : FunctionClosure

i3 : matrix{{2}} / 2

o3 = | 1 |

              1       1
o3 : Matrix QQ  <-- QQ
```

Reply:
I'm proposing that we define them.  :)  I think the result being over the fraction field seems like reasonable behavior.  That's how `ZZ / ZZ` and `RingElement / RingElement` work.

Reply:
I guess my point is that using `1/r * m` instead makes it clear that you _should_ expect something over the fraction field, so maybe at least caveat this in the documentation.